src/natcap/invest/ndr/ndr.py

```diff
@@ -646,13 +651,32 @@ def _validate_inputs(nutrients_to_process, lucode_to_parameters):
             base_raster_list, aligned_raster_list,
             ['near']*len(base_raster_list), dem_info['pixel_size'],
             'intersection'),
-        kwargs={
-            'base_vector_path_list': [args['watersheds_path']],
-            'vector_mask_options': {
-                'mask_vector_path': args['watersheds_path']}},
+        kwargs={'base_vector_path_list': [args['watersheds_path']]},
         target_path_list=aligned_raster_list,
         task_name='align rasters')
 
+    mask_runoff_dependent_task = align_raster_task
+    if pygeoprocessing.get_raster_info(
+            f_reg['aligned_runoff_proxy_path'])['nodata'][0] is None:
+        mask_runoff_dependent_task = task_graph.add_task(
+            func=_assign_nodata,
+            kwargs=dict(
```

Comment:
I see you use `dict()` here and `{ }` below for kwargs syntax, I don't think there's an issue either way but maybe good to be consistent?

Reply:
Replaced with `{ }`